tools/public_api_guard/material/snack-bar.d.ts

```diff
@@ -92,3 +105,11 @@ export declare class SimpleSnackBar {
     static ɵcmp: i0.ɵɵComponentDefWithMeta<SimpleSnackBar, "simple-snack-bar", never, {}, {}, never, never>;
     static ɵfac: i0.ɵɵFactoryDef<SimpleSnackBar, never>;
 }
+
+export interface TextOnlySnackBar {
```

Comment:
Thinking about this more, could we prefix the newly added interfaces with an underscore? Maybe worth discussing in the team meeting, but I want to start making it more clear that some symbols are internal aside from just not documenting them.